content/sensu-core/1.6/reference/checks.md

```diff
@@ -347,6 +348,36 @@ outputting the last 100 lines of an error log.
 }
 {{< /highlight >}}
 
+## Managing checks
+
+### Delete a check
+
+To delete a check from Sensu, remove the configuration file for the check, then restart the Sensu server and API.
```

Comment:
Do we know if this endpoint supports standalone checks? I want to say no.

If they are supported, we need to call out that the Sensu client where the check definition would need restarted.
If they are not supported, we should call out that this won't work for stand alone checks either here or possibly here and/or elsewhere.

My lab is down at the moment. I'll work on bringing it up to test.